<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GESTIONNAIREMARKTING</class>
 <widget class="QDialog" name="GESTIONNAIREMARKTING">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Publicite">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>681</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>PLAN</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_description">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>401</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>68</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DESCRIPTION</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_ajoutpaln">
        <property name="text">
         <string>AJOUTER PLAN</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/Nouveau dossier/check.png</normaloff>:/new/prefix1/Nouveau dossier/check.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_suppplan">
        <property name="text">
         <string>SUPPRIMER PLAN</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/Nouveau dossier/button_cancel.png</normaloff>:/new/prefix1/Nouveau dossier/button_cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_afficher_plan">
        <property name="text">
         <string>AFFICHER PLAN</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_controlerpub">
        <property name="text">
         <string>CONTROLER PUB</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_distinataire">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>38</y>
       <width>69</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DISTINATAIRE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>29</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TITRE</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>-100</y>
       <width>871</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix3/amine/shutterstock_83741023-300x196.jpg);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_distinataire">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>38</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_plantitre">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>12</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>graphicsView</zorder>
    <zorder>tableView</zorder>
    <zorder>lineEdit_description</zorder>
    <zorder>label_2</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>label_distinataire</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_distinataire</zorder>
    <zorder>lineEdit_plantitre</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>DEMANDE</string>
    </attribute>
    <widget class="QTableView" name="tableView_demande">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>431</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-60</y>
       <width>681</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix3/amine/shutterstock_83741023-300x196.jpg);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>225</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>AFFICHER DEMANDE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</normaloff>:/new/prefix1/Nouveau dossier/book_256px.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajoutdemande">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>126</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>AJOUTER DEMANDE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/prefix1/Nouveau dossier/check.png</normaloff>:/new/prefix1/Nouveau dossier/check.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supdemande">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>159</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>SUPPRIMER DEMANDE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/prefix1/Nouveau dossier/button_cancel.png</normaloff>:/new/prefix1/Nouveau dossier/button_cancel.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifierdemande">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>192</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>MODIFIER DEMANDE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/prefix1/Nouveau dossier/applications_development.png</normaloff>:/new/prefix1/Nouveau dossier/applications_development.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_demande">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>31</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_de_lapplication">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>91</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>31</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_DEMANDE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>61</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM DU CLIENTS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_du_clients">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>61</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>91</y>
       <width>108</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TYPE DE L'PPLICATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>121</y>
       <width>145</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LANQUAGE DE L'APPLIACTION</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lanquage">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>121</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>151</y>
       <width>25</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="TimeEdit_datee">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>151</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>graphicsView_2</zorder>
    <zorder>lineEdit_id_demande</zorder>
    <zorder>lineEdit_nom_du_clients</zorder>
    <zorder>lineEdit_type_de_lapplication</zorder>
    <zorder>lineEdit_lanquage</zorder>
    <zorder>TimeEdit_datee</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_5</zorder>
    <zorder>tableView_demande</zorder>
    <zorder>pushButton_ajoutdemande</zorder>
    <zorder>pushButton_supdemande</zorder>
    <zorder>pushButton_modifierdemande</zorder>
    <zorder>pushButton</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix3/amine/shutterstock_83741023-300x196.jpg);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>Publicite</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
